ase/rtl/CMakeLists.txt

```diff
@@ -0,0 +1,153 @@
+
+set(ASE_PKG_FILE ${ASE_SERVER_RTL}/ase_pkg.sv)
+set(PLATFORM_PKG_FILE ${PLATFORM_IF_RTL}/ccip_if_pkg.sv)
+
+if(ASE_SIMULATOR STREQUAL "QUESTA")
+  find_package(Quartus)
+  find_package(Questa)
+  set(ASE_WORKDIR             "${PROJECT_BINARY_DIR}")
+  set(ASE_CONFIG              "${PROJECT_BINARY_DIR}/ase.cfg")
+  set(ASE_SCRIPT              "${PROJECT_BINARY_DIR}/ase_regress.sh")
+  set(ALTERA_MEGAFUNCTIONS    "${QUARTUS_DIR}/../modelsim_ae/altera/verilog/altera_mf" )
+
+  # Create ASE scripts
+  configure_file(${CMAKE_SOURCE_DIR}/in/ase.cfg.in
+    ${PROJECT_BINARY_DIR}/ase.cfg)
+  configure_file(${CMAKE_SOURCE_DIR}/in/ase_regress.sh.in
+    ${PROJECT_BINARY_DIR}/ase_regress.sh)
+  configure_file(${CMAKE_SOURCE_DIR}/in/vsim_run.tcl.in
+    ${PROJECT_BINARY_DIR}/vsim_run.tcl)
+
+  set(questa_flags "")
+  list(APPEND questa_flags +define+VENDOR_ALTERA)
+  list(APPEND questa_flags +define+TOOL_QUARTUS)
+  list(APPEND questa_flags +define+${ASE_SIMULATOR})
+  list(APPEND questa_flags +define+${ASE_PLATFORM})
+  set(QUESTA_VLOG_DEFINES "${questa_flags}"
+    CACHE STRING "Modelsim/Questa compiler define flags")
+
+  set(questa_flags "")
+  list(APPEND questa_flags +incdir+.+work+${ASE_SERVER_RTL}+${PLATFORM_IF_RTL})
+  set(QUESTA_VLOG_INCLUDES "${questa_flags}"
+    CACHE STRING "Modelsim/Questa compiler include flags")
+
+  set(questa_flags "")
+  list(APPEND questa_flags -nologo -sv +librescan)
+  list(APPEND questa_flags -timescale ${ASE_TIMESCALE})
+  list(APPEND questa_flags -work work)
+  list(APPEND questa_flags -novopt)
+  list(APPEND questa_flags ${QUESTA_VLOG_DEFINES})
+  list(APPEND questa_flags ${QUESTA_VLOG_INCLUDES})
+  set(QUESTA_VLOG_FLAGS "${questa_flags}"
+    CACHE STRING "Modelsim/Questa compiler flags")
+
+  # SW library name
+  set(ASE_SHOBJ_NAME "libopae-c-ase-server")
+  set(ASE_SHOBJ_SO  ${ASE_SHOBJ_NAME}.so)
+
+  set(questa_flags "")
+  list(APPEND questa_flags -novopt)
+  list(APPEND questa_flags -c)
+  list(APPEND questa_flags -dpioutoftheblue 1)
+  # list(APPEND questa_flags -wlf ${questa_waveform})
+  list(APPEND questa_flags -sv_lib ${CMAKE_BINARY_DIR}/sw/${ASE_SHOBJ_NAME})
+  list(APPEND questa_flags -do vsim_run.tcl)
+  list(APPEND questa_flags -sv_seed 1234)
+  list(APPEND questa_flags -L ${ALTERA_MEGAFUNCTIONS})
+  list(APPEND questa_flags -l vlog_run.log)
+  string(REPLACE ";" " " questa_flags "${questa_flags}")
+  set(QUESTA_VSIM_FLAGS "${questa_flags}"
+    CACHE STRING "Modelsim/Questa simulator flags")
+
+  # Define rule to generate platform_afu_top_config.vh
+  set(questa_flags "")
+
+  # afu_platform_config --sim --tgt=rtl --src ccip_std_afu.json  intg_xeon
+  file(MAKE_DIRECTORY ${PROJECT_BINARY_DIR}/platform_includes)
+  add_custom_command(OUTPUT "platform_includes/platform_afu_top_config.vh"
+    WORKING_DIRECTORY ${PROJECT_BINARY_DIR}
+    COMMAND ${AFU_PLATFORM_CONFIG} --sim --tgt=platform_includes --src ${CMAKE_BINARY_DIR}/ccip_std_afu.json ${ASE_PLATFORM_ABBREV})
```

Comment:
I assume this reference to CMAKE_BINARY_DIR/ccip_std_afu.json is wrong, but don't really understand the code. The JSON comes from the sources.